worker/gouging.go

```diff
@@ -272,7 +272,10 @@ func checkPriceGougingPT(gs api.GougingSettings, cs api.ConsensusState, txnFee t
 	}
 
 	// check LatestRevisionCost - expect sane value
-	maxRevisionCost := gs.MaxRPCPrice.Add(gs.MaxDownloadPrice.Div64(1 << 40).Mul64(2048))
+	maxRevisionCost, overflow := gs.MaxRPCPrice.AddWithOverflow(gs.MaxDownloadPrice.Div64(1 << 40).Mul64(2048))
```

Comment:
The UI says these are set in TB, but the doc string says TiB. We should follow up and make sure the units are consistent everywhere.

Is there any reason not to store the gouging settings in the base unit, per byte and per byte/block? That removes any ambiguity and lets clients convert it how they see fit.

Reply:
Yeah we will probably update that in v2